### DISCUSSION TITLE
How to limit maximum upload size ?

### DISCUSSION THREAD

**Discussion by pitagora04** [torchbox/wagtailmedia#197] [Q&A]
Hi there, I'm just starting to use this package. It seems to be very nice.
I have a question: how should I limit maximum upload size for a Video and/or Audio ?
I want to allow only small videos up to 100MB size.
Thanks !
EDIT: I see `WagtailImageField` has `max_upload_size` which gets max upload size from `WAGTAILIMAGES_MAX_UPLOAD_SIZE` setting. But I don't find something similar here in **wagtailmedia**...

**Comment by zerolab** [comment]
🤔 We don't have a mechanism for this yet https://github.com/torchbox/wagtailmedia/blob/main/src/wagtailmedia/models.py#L33

I've created an issue to track this
